### PR TITLE
Added Admin Features on The Dashboard

### LINKED ISSUE #8839: Add Admin Features on The Dashboard
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [Header File](https://github.com/publiclab/plots2/blob/main/app/views/layouts/_header.html.erb) in the `https://github.com/publiclab/plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Include a link to the features page for the admin within this file `app/views/layouts/_header.html.erb`
after line 135

https://github.com/publiclab/plots2/blob/f82a11c17404bae7b4307313b6890ca3169e8c90/app/views/layouts/_header.html.erb#L135

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
 <a class="dropdown-item" href="/people">Contributors</a>
+<a class="dropdown-item" href="/features">Features</a>
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by RATED-R-SUNDRAM**
Fixes #8839  

Added the Admin feature on the Dashboard

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -132,7 +132,7 @@
                       <a class="dropdown-item" href="/useremail">Email search</a>
                       <a class="dropdown-item" href="/wiki/locked">Locked pages</a>
                       <a class="dropdown-item" href="/spam">Spam moderation</a>
-                      <a class="dropdown-item" href="/people">Contributors</a>
+                      <a class="dropdown-item" href="/features">Features</a>
```
Comment (RuthNjeri) [inline app/views/layouts/_header.html.erb:135]:
```suggestion
                      <a class="dropdown-item" href="/people">Contributors</a>
                      <a class="dropdown-item" href="/features">Features</a>
```

Thanks for opening this up @RATED-R-SUNDRAM 
Just a slight correction, the Contributors link should not be removed...the Features link should be added...sorry for any confusion caused by the issue...

**Review by RuthNjeri — APPROVED** [review]
Thanks @RATED-R-SUNDRAM, the changes look good!

Just a slight comment, whenever you work on a task, you should create a new branch which describes the work you are doing for example, instead of  making the changes on the  `RATED-R-SUNDRAM:main` you can create a new branch from the main branch...

have a look at this blog, step 4 https://medium.com/@jenweber/your-first-open-source-contribution-a-step-by-step-technical-guide-d3aca55cc5a6

**Review by cesswairimu — APPROVED** [review]
Awesome :tada:

**Comment by cesswairimu** [comment]
Thanks @RATED-R-SUNDRAM  for working on this :tada:  and thanks @RuthNjeri  for the reviews :tada: :tada:

**Pull request merged**